conf.d/done.fish

```diff
@@ -258,7 +258,7 @@ if set -q _done_enabled
 
                 # make notification auto-disappear
                 set -l transient ""
-                if test "$__done_notification_transient" -eq 1
+                if set -q __done_notification_transient; and test "$__done_notification_transient" -eq 1
```

Comment:
Just compare it as string instead of number; a string will always be there (even if it resolves to `""`). Also, undo the bug introduced in https://github.com/franciscolourenco/done/pull/116, which changed the default behavior from being transient to being non-transient (transient should be the default as requested by author in https://github.com/franciscolourenco/done/pull/102#issuecomment-799348676).

```suggestion
                if test "$__done_notification_transient" != 0
```